luatest/server.lua

```diff
@@ -328,19 +350,30 @@ function Server:restart(params, opts)
     log.debug('Restarted server PID: ' .. self.process.pid)
 end
 
+-- Save server artifacts by copying the working directory
+-- Throws an error when the copying is not successful.
+function Server:save_artifacts()
```

Comment:
You expose this function as a part of the public API now. Do you really want this?

Reply:
Fixed :rocket: